pkg/versioning/helm_test.go

```diff
@@ -165,6 +165,21 @@ func TestHelmChartSetVersion(t *testing.T) {
 		assert.Equal(t, "1.2.4", helmChart.metadata.AppVersion)
 	})
 
+	t.Run("success case - update app version (semver2)", func(t *testing.T) {
```

Comment:
Should we add some info related to replacing + sign in name of test case?

Reply:
Done - [ad2835d](https://github.com/SAP/jenkins-library/pull/3980/commits/ad2835d442d9c438dfe52c425ea4ef9090a7630a)